src/driver-asus.c

```diff
@@ -443,6 +443,11 @@ asus_driver_probe(struct ratbag_device *device)
 	if (rc)
 		return rc;
 
+	/* HID device descriptor must have input and output */
+	if (!ratbag_hidraw_has_report(device, HID_INPUT_REPORT) ||
+	    !ratbag_hidraw_has_report(device, HID_OUTPUT_REPORT))
+		return -1;
```

Comment:
Could you change this to `-ENODEV`?
The `probe` function is supposed to only return POSIX error codes on error, moreover, if it's something other than ENODEV it's going to print it to stderr.